ida/data.yml

```diff
@@ -10894,18 +11034,19 @@ classes:
       - ea: 0x1421FF140
         pointer: False
     funcs:
-      0x1408E8660: ctor
-      0x1408E8710: dtor
+      0x1408E8660: Create
+      0x1408E8710: Destroy
```

Comment:
Should still be named `dtor`

Reply:
This also is not the dtor. The dtor is part of the vtable (@ 0x141A6A198).